Resolutions/XenobladeX_Resolution/rules.txt

```diff
@@ -3,169 +3,230 @@ titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
-$scaleShader = 1.0 #(1440.0/1080.0)
-$scaleBlur = 0.5
 
 [Preset]
-name = 2560x1080 (21:9 HD)
+name = 1440p
+category = Resolution
 $width = 2560
-$height = 1080
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 1.0
-$dither = 0.15
-$scaleShader = (1440.0/1080.0)
-$scaleBlur = 0.5
+$height = 1440
 
 [Preset]
-name = 1920x1200 (10:9 HD)
-$width = 1920
-$height = 1200
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 1.0
-$dither = 0.15
-$scaleShader = (1440.0/1200.0)
-$scaleBlur = 0.5
+name = 2160p (4k)
+category = Resolution
+#$width = 3840
+#$height = 2160
 
+[Preset]
+name = 4320p (8k)
+category = Resolution
+#$width = 7680
+#$height = 4320
 
 [Preset]
-name = 2560x1440
-$width = 2560
-$height = 1440
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 0.75
-$dither = 0.15
-$scaleShader = (1440.0/1080.0)
-$scaleBlur = 0.5
+name = 720x480
+category = Resolution
+$width = 720
+$height = 480
+
+[Preset]
+name = 1360x768
+category = Resolution
+$width = 1360
+$height = 768
 
+[Preset]
+name = 1600x900
+category = Resolution
+$width = 1600
+$height = 900
+
+[Preset]
+name = 2560x1080
+category = Resolution
+$width = 2560
+$height = 1080
 
 [Preset]
-name = 3440x1440 (21:9)
+name = 3440x1440
+category = Resolution
 $width = 3440
 $height = 1440
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 1.0
-$dither = 0.1
-$scaleShader = 1.0
-$scaleBlur = 0.5
 
 [Preset]
 name = 3840x1620 (21:9 4k panel 1:1 pix mapping, transition fix)
+category = Resolution
 $width = 3840
 $height = 1620
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 0.5
-$dither = 0.25
-$scaleShader = (1440.0/1620.0) #round 1. shadow to 2.0
-$scaleBlur = 1.0
-
 
 [Preset]
 name = 3200x1800
+category = Resolution
 $width = 3200
 $height = 1800
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 2.0
-$dither = 0.2
-$scaleShader = (1440.0/1800.0)
-$scaleBlur = 0.5
-
-
-[Preset]
-name = 3840x2160
-$width = 3840
-$height = 2160
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 0.5
-$dither = 0.25
-$scaleShader = (1440.0/1080.0) #round 1.5 shadow to 2.0
-$scaleBlur = 1.0
 
 [Preset]
 name = 5120x2160 (4k 21:9)
+category = Resolution
 $width = 5120
 $height = 2160
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 0.5
-$dither = 0.2
-$scaleShader = (1440.0/1080.0) #round 1.5 shadow to 2.0
-$scaleBlur = 1.0
-
 
 [Preset]
 name = 5120x2880
+category = Resolution
 $width = 5120
 $height = 2880
-$gameWidth = 1280
-$gameHeight = 720
-$internalRes = 0.375
-$dither = 0.25
-$scaleShader = (1440.0/1080.0)
-$scaleBlur = 1.0
 
-#[Preset]
+[Preset]
 #name = 7680x4320
-#$width = 7680
-#$height = 4320
-#$gameWidth = 1280
-#$gameHeight = 720
-#$internalRes = 0.5
-#$dither = 0.25
-#$scaleShader = 1.0
-#$scaleBlur = 1.0
-#
-#[Preset]
-#name = 10240x5760
-#$width = 10240
-#$height = 5760
-#$gameWidth = 1280
-#$gameHeight = 720
-#$internalRes = 0.5
-#$dither = 0.25
-#$scaleShader = 1.0
-#$scaleBlur = 1.0
+category = Resolution
+$width = 7680
+$height = 4320
+
+[Preset]
+name = 10240x5760
```

Comment:
Do we even need this high of a resolution? Not even the high-end Turing GPUs can handle this at an okay framerate.

Reply:
I removed this after following the BOTW resolutions.